backend/src/main/java/com/example/backend/use_case/display_stops/StopsInteractor.java

```diff
@@ -0,0 +1,32 @@
+package com.example.backend.use_case.display_stops;
+
+import com.example.backend.data_access.stop.StopDataAccessInterface;
+import com.example.backend.entity.Location;
+
+import java.util.HashMap;
+
+public class StopsInteractor implements StopsInputBoundary {
+    final StopDataAccessInterface stopDataAccessInterface;
```

Comment:
should be named stopDataAccessObject. The interface exists to conform to Dependency Inversion Principle, and so therefore the type is correct, but remember that the variable is no longer an interface but rather an object.

Reply:
Done.